srw_detector.py

```diff
@@ -28,18 +29,22 @@ class SRWDetector(Device):
     horizontal_extent = Cpt(Signal)
     vertical_extent = Cpt(Signal)
 
-    def __init__(self, name, motor0, field0, motor1, field1, reg=None,
-                 sim_id=None, sirepo_server='http://10.10.10.10:8000',
+    def __init__(self, name, cname, spec_name1, spec_name2, motor0, field0, motor1=None, field1=None, reg=None,
```

Comment:
A few comments here:
- `cname` is not clear, can you make a self-explanatory name please?
- `spec_name1` and `spec_name1` don't match the naming scheme of `motor`s and `field`s, I'd recommend to start the counter from 0 to follow the existing convention.
- Please add a docstring with explanation of the parameters.

Reply:
Naming changed / docstring added